packages/scripts/utils/config.js

```diff
@@ -281,7 +283,7 @@ function getWebpackEntryPoints( buildType ) {
 								) }". File is located outside of the "${ getWordPressSrcDirectory() }" directory.`
 							)
 						);
-						return;
+						continue;
```

Comment:
@sirreal, I checked the previous implementation before the function has support for `buildType` added in https://github.com/WordPress/gutenberg/pull/57461 and the `return` was inside the callback rather than loop. So it should continue execution and skip only misconfigured entries.

Reply:
I think you're correct, these `continue`s look like bugfixes.

Let's extract this change to its own PR.

Reply:
Excellent idea. While I played with it, I discovered more issues that I fixed.